src/content/docs/en/guides/environment-variables.mdx

```diff
@@ -79,7 +79,7 @@ You can also add environment variables as you run your project:
 
 ## Getting environment variables
 
-Instead of using `process.env`, with Vite you use `import.meta.env`, which uses the `import.meta` feature added in ES2020.
+Instead of using `process.env`, you use `import.meta.env`, which uses the `import.meta` feature added in ES2020.
```

Comment:
```suggestion
Instead of relying on `process.env` for environment variables, use `import.meta.env`. This approach leverages the `import.meta` feature, introduced in the ES2020 specification.
```

Reply:
What about:

> Environment variables in Astro are accessed with `import.meta.env`, using the [`import.meta` feature added in ES2020](https://tc39.es/ecma262/2020/#prod-ImportMeta), instead of `process.env`.

Reply:
That's even better.

Reply:
Go ahead and update your suggestion, @dreyfus92 , and I'll commit it!